graylog2-server/src/main/java/org/graylog2/plugin/lookup/LookupDataAdapter.java

```diff
@@ -84,6 +107,21 @@ public void stop() {
     }
     protected abstract void doStop() throws Exception;
 
+    /**
+     * Returns the refresh interval for this data adapter. Use {@link Duration.ZERO} if refresh should be disabled.
```

Comment:
extreme nitpick: javadoc links to members must use '#' instead of '.', otherwise IDEs will flag an error:
`{@link Duration#ZERO}`